modules/shortcodes/tweet.php

```diff
@@ -80,6 +80,17 @@ public static function jetpack_tweet_shortcode( $atts ) {
 			}
 		}
 
+		if ( class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() ) {
+			$width  = ! empty( $attr['width'] ) ? $attr['width'] : 600;
+			$height = 480;
```

Comment:
The default width and height are [from the AMP plugin](https://github.com/ampproject/amp-wp/blob/develop/includes/embeds/class-amp-base-embed-handler.php#L19-L26).